spack/repo/packages/py-pyprecice/package.py

```diff
@@ -18,7 +18,7 @@ class PyPyprecice(PythonPackage):
     maintainers = ["ajaust", "BenjaminRueth"]
```

Comment:
Update `BenjaminRueth` to `BenjaminRodenberg`

Reply:
I think this change is already applied upstream in https://github.com/spack/spack/pull/22012. I will take care of synchronizing upstream with `python-bindings/develop`, then merge `develop` into this PR. This should resolve this point.

Reply:
Resolved by https://github.com/precice/python-bindings/pull/89/commits/8ee7c7665f7538d99d481ebf9c067063501718db